reactor-test/src/main/java/reactor/test/StepVerifier.java

```diff
@@ -1072,10 +1072,17 @@ static void resetDefaultTimeout() {
 		Assertions hasDiscardedExactly(Object... values);
 
 		/**
-		 * Assert that the tested publisher has discarded at least one element matching the given
-		 * predicate to the {@link reactor.core.publisher.Flux#doOnDiscard(Class, Consumer) discard} hook.
+		 * Assert that the tested publisher has discarded one or more errors to the
+		 * {@link reactor.core.publisher.Flux#doOnDiscard(Class, Consumer) discard} hook,
+		 * and check that the collection of discarded elements matches a predicate.
 		 */
-		Assertions hasDiscardedMatching(Predicate<Object> matcher);
+		Assertions hasDiscardedElementsMatching(Predicate<Collection<Object>> matcher);
+
+		/**
+		 * Assert that the tested publisher has discarded one or more elements matching to the
```

Comment:
remove the "matching" at the end of the line

Reply:
done